Makefile

```diff
@@ -77,32 +85,35 @@ ifdef GOBIN
 TANZU_CLI_INSTALL_PATH = "$${GOBIN}/tanzu"
 endif
 
-#INSTALLED_CLI_DIR
-
 ifeq ($(TCE_CI_BUILD), true)
 XDG_DATA_HOME := /tmp/mylocal
+XDG_CACHE_HOME := /tmp/mycache
+XDG_CONFIG_HOME := /tmp/myconfig
 SED := sed -i
 endif
 ifeq ($(XDG_DATA_HOME),)
 ifeq ($(build_OS), Darwin)
 XDG_DATA_HOME := "${HOME}/Library/Application Support"
+XDG_CACHE_HOME := ${HOME}/.cache
+XDG_CONFIG_HOME := ${HOME}/.config
 SED := sed -i '' -e
 endif
 endif
 ifeq ($(XDG_DATA_HOME),)
 ifeq ($(build_OS), Linux)
 XDG_DATA_HOME := ${HOME}/.local/share
+XDG_CACHE_HOME := ${HOME}/.cache
+XDG_CONFIG_HOME := ${HOME}/.config
 SED := sed -i
 endif
 endif
 
 export XDG_DATA_HOME
+export XDG_CACHE_HOME
+export XDG_CONFIG_HOME
 export GO
 export GOLANGCI_LINT
 export ARTIFACTS_DIR
-
-PRIVATE_REPOS="github.com/vmware-tanzu/*"
```

Comment:
Thanks, I've been meaning to get rid of this.

Reply:
yea.... needed a clean up :)